assets/erc-7765/contracts/ERC7765Example.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Code in your assets directory must be provided under a relatively permissive license. We prefer CC0-1.0, but also accept MIT or Apache-2.0. We cannot accept GPL or other copyleft licenses.